tools/vhost-user-video/main.c

```diff
@@ -1241,6 +1241,88 @@ struct virtio_video_get_control_resp_bitrate {
     struct virtio_video_control_val_bitrate bitrate;
 };
 
+static int
+handle_query_control_cmd(struct VuVideo *v,
+                         struct vu_video_ctrl_command *cmd)
```

Comment:
I would consider adding a few sentences documenting what's the expected return and error code.
Doesn't sound like it needs deep documentation as doesn't look as anything to be used as API but it will help when getting back to the function.

Reply:
Yes, that sounds reasonable. Other virtio devices and similar functions within this patch lack documentation, and some comments would be welcome for maintainers.

I will skip it for this PR, but I have another task planned to refactor the code, where I should add some documentation where needed (specially for header files). Thanks for taking a look :)